File: java/src/sorting_algorithms/BubbleSort.java
package sorting_algorithms;


public class BubbleSort {

    public static int[] bubbleSort(int[] arr){
        int temp;
        for(int i = arr.length; i > 0; i--)
            for (int j = 0; j < i-1; j++) {
                if(arr[j] > arr[j+1]){
                    temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }
            }
        return arr;
    }
}

File: python/algorithms/sorting/bubble_sort.py
def bubble_sort(arr):
    i = 0
    while i < len(arr):
        j = i+1
        while j < len(arr):
            if arr[i] > arr[j]:
                arr[j], arr[i] = arr[i], arr[j]
            j += 1
        i += 1
    return arr
